src/main/java/de/tum/in/www1/artemis/domain/GuidedTourSettings.java

```diff
@@ -0,0 +1,50 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.io.Serializable;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.Table;
+
+/**
+ * GuidedTourSettings
+ **/
+@Embeddable
+@Table(name = "guided_tour_settings")
+public class GuidedTourSettings implements Serializable {
```

Comment:
I think this implementation is better. What I still wonder: What happens if a guided tour changes after it was introduced, e.g. when a new feature has been developed and the guided tour needs one additional step in the middle?